lib/src/Base/Geom/openturns/Domain.hxx

```diff
@@ -52,6 +52,12 @@ public:
   /** Check if the closed domain contains given points */
   virtual BoolCollection contains(const Sample & sample) const;
 
+  /** Compute the Euclidean distance from a given point to the domain */
+  virtual Scalar computeDistanceToDomain(const Point & point) const;
+
+  /** Compute the Euclidean distance from given points to the domain */
+  virtual Sample computeDistanceToDomain(const Sample & sample) const;
+
```

Comment:
This method name would suggest that we compute a distance from a domain (as class is Domain)  to another domain (as argument)
but from the class point of view it is actually a distance to point(s) as we'll likely write something like:
```domain.computeDistance(point)```
So I would suggest just 'computeDistance' (or computeDistanceToPoint)

Reply:
OK, point taken. Do you think the name of the class `DistanceToDomainFunction` should also be changed?

Reply:
this one is fine, no need to explicit that it is a domain-point distance

another possibility would be 'DomainDistanceFunction'